requirements.txt

```diff
@@ -0,0 +1,7 @@
+pandas==1.2.3
```

Comment:
The requirements.txt file with the main Opni repository must be modified to have these additional libraries.

Reply:
Thanks to @AmartC  this has been addressed in this PR: https://github.com/rancher/opni/pull/104